doc/_static/css/custom.css

```diff
@@ -77,3 +77,14 @@ h6,
   font-weight: unset;
   line-height: unset;
 }
+
+/*Roseau colors */
```

Comment:
With these css properties, we can now reference and use the colors in our documentation